src/display/canvas.js

```diff
@@ -1419,6 +1416,9 @@ class CanvasGraphics {
 
   // Graphics state
   setLineWidth(width) {
+    if (width !== this.current.lineWidth) {
+      this._cachedScaleForStroking = null;
```

Comment:
Is it not necessary to reset *both* caches here?

Reply:
The other cached value doesn't use the lineWidth, so it's useless.